.github/ISSUE_TEMPLATE/Release_23.12_FOSS_ComplianceTemplate.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Release Compliance 23.12
+about: Use this template for tracking Catena-X Release 23.12.
```

Comment:
Use this template to track all Compliance-relevant topics for your component with regards to the upcoming Milestone.